### ISSUE TITLE
Concept name problem, HP:0025480

### ISSUE THREAD

**Issue by mirandajarnot** [monarch-initiative/HPO-UMLS-Integration#40]
In the June 28, 2017 version of HPO, there is a term name missing for HP:0025480, and it has been replaced with the term's definition.  
The term name has become "A type of spinal dysraphism presenting as a subcutaneous fatty mass, that is, a spinal defect associated with lipomatous tissue, and covered by skin. The most usual location for lipomyelomeningocele is at the gluteal cleft." 
Can you please let us know what the correct name of this term is supposed to be?  This error is also present in the context hierarchy.   The parent term is Myelomeningocele.

**Comment by pnrobinson** [comment]
Thanks for picking this up. The term name (label) should be

Lipomyelomeningocele


 I have just corrected this on our github repo.

-Peter


Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________
